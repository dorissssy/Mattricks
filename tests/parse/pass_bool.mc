function f () gives bool {
    a = bool true;
    return a;
}