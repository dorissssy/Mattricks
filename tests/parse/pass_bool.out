

Parsed program: 


bool f()
{
a = bool true;
b := true;
c = bool 0;
return a;
}

